app/src/main/java/nl/ags/picum/dataStorage/managing/AppDatabaseManager.java

```diff
@@ -10,6 +10,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import kotlin.NotImplementedError;
```

Comment:
@Jesse592 ???? Kotlin??

Reply:
Not sure why this is in the database class, however the OkHttp dependency is from Kotling so seeing some kotlin imports is normal (Should only be in the RouteCalculator class)

Reply:
@JustRostin Now fixed